nexus-sdk-js/src/utils/http.ts

```diff
@@ -63,13 +103,23 @@ const parseResponse = (response: Response, parser = jsonParser): any => {
   }
 };
 
-export function httpGet(url: string, useBase: boolean = true): Promise<any> {
+const parseJsonError = async (error: Response): Promise<JSON> => {
+  const payload: JSON = await error.json();
+  return payload;
+};
+
+const GET_DEFAULT_HTTP_CONFIG = {
```

Comment:
Nitpicking: the meaning would be clearer if it were called CONFIG_DEFAULT_HTTP_GET, and the next one CONFIG_DEFAULT_HTTP_POST

Reply:
👍